sdk/Cargo.toml

```diff
@@ -11,13 +11,14 @@ crate-type = ["cdylib", "lib"]
 name = "light_sdk"
 
 [features]
+anchor = []
 no-entrypoint = []
 no-idl = []
 no-log-ix-name = []
 cpi = ["no-entrypoint"]
 custom-heap = ["light-heap"]
```

Comment:
what do we need `light-heap` for in `light-sdk`?
can we remove `light-heap` and the custom heap feature from this cargo toml?
(one feedback from squads was that they use their own heap implementation and encountered conflicts.)